src/Stratis.FederatedPeg.Features.FederationGateway/RestClients/RestApiClientBase.cs

```diff
@@ -53,12 +63,19 @@ public RestApiClientBase(ILoggerFactory loggerFactory, IFederationGatewaySetting
                     // Retry the following call according to the policy.
                     await this.policy.ExecuteAsync(async token =>
                     {
-                        this.logger.LogDebug("Sending request of type '{0}' to Uri '{1}'.", requestModel.GetType().FullName, publicationUri);
+                        this.logger.LogDebug("Sending request of type '{0}' to Uri '{1}'.",
+                            requestModel.GetType().FullName, publicationUri);
 
-                        response = await client.PostAsync(publicationUri, request).ConfigureAwait(false);
+                        response = await client.PostAsync(publicationUri, request, cancellation).ConfigureAwait(false);
                         this.logger.LogDebug("Response received: {0}", response);
 
-                    }, CancellationToken.None);
+                    }, cancellation);
+                }
+                catch (OperationCanceledException)
+                {
+                    this.logger.LogError("Operation canceled.");
+                    this.logger.LogTrace("(-)[CANCELED]:null");
```

Comment:
```suggestion
                     this.logger.LogTrace("(-)[CANCELLED]:null");
```

Changed for consistency + more British.

Reply:
fixed